assets/src/util/sentry.tsx

```diff
@@ -27,8 +86,12 @@ const initSentry = (appString: string) => {
   const { sentry: sentryDsn, environmentName: env } = getDataset();
 
   if (sentryDsn && isRealScreen()) {
+    Raven.setTransport(customTransport);
     Raven.config(sentryDsn, { environment: env }).install();
 
+    // Set listener to retry any cached Sentry events any time the browser transitions from offline to online
+    window.addEventListener("online", resendCachedEvents);
```

Comment:
The official transport seems to also have some retry logic built-in, so if the browser never technically goes "offline" by its own definition (and so this event never fires), but there was still a sending error at some point, the queued events would eventually be retried. Maybe a simple timer that ticks every 5 minutes or so?